idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -6,9 +6,9 @@ export default `
         <div class="details-header">
           <h1 data-hook="title"></h1>
           <div class="idea-controls">
-            <button data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
-            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span class="transition-text">Disabling...</span><span class="default-text">Disable <span data-hook="title"></span></span></button>
-            <button data-hook="install" id="install-button" class="button primary"><span class="state-change-inner"></span><span class="transition-text">Enabling...</span><span class="default-text">Enable <span data-hook="title"></span></span></button>
+            <button data-l10n-id="giveFeedback" data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
+            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span data-l10n-id="disableExperimentTransition" class="transition-text">Disabling...</span><span data-l10n-id="disableExperiment" class="default-text">Disable <span data-hook="title"></span></span></button>
```

Comment:
Not actually hacking around HTML localization here, FWIW. We don't actually need the embedded HTML in this string, it's just there as a DOM hook for population by Ampersand. But, L20N does that for us now, instead.